projects/plugins/jetpack/_inc/client/at-a-glance/security-bundle/index.jsx

```diff
@@ -0,0 +1,131 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { getRedirectUrl } from '@automattic/jetpack-components';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import Button from 'components/button';
+import Card from 'components/card';
+import bundleImageUrl from './shield-with-check.svg';
+import cloudImageUrl from './cloud.svg';
+import shieldImageUrl from './shield.svg';
+import removeBugImageUrl from './remove-bug.svg';
+import { getProductDescriptionUrl } from 'product-descriptions/utils';
+import { isFetchingSitePurchases as getIsFetchingSitePurchases, hasActiveSecurityPurchase as getHasActiveSecurityPurchase } from 'state/site';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class DashSecurityBundle extends Component {
+	trackUpgradeClick = () => {
+		analytics.tracks.recordJetpackClick( {
+			page: 'aag',
+			target: 'upgrade-button',
+			feature: 'security',
+		} );
+	};
+
+	renderFeatures() {
+		const features = [
+			{
+				id: 'backup',
+				imageUrl: cloudImageUrl,
+				text: __( 'Backup', 'jetpack' ),
+				imgAlt: __( 'A cloud representing Jetpack Backup', 'jetpack' ),
+			},
+			{
+				id: 'scan',
+				imageUrl: shieldImageUrl,
+				text: __( 'Scan', 'jetpack' ),
```

Comment:
I would suggest using `_x` here, so we can indicate that those are not verbs, but product names.